packages/core/src/generators/vue/compositionApi.ts

```diff
@@ -51,8 +51,10 @@ export function generateCompositionApiScript(
     functions: false,
     getters: false,
     format: 'variables',
-    valueMapper: (code, _, typeParameter) =>
-      isTs && typeParameter ? `ref<${typeParameter}>(${code})` : `ref(${code})`,
+    valueMapper: (code, _, typeParameter) => {
+      const cleanCode = code.replaceAll('this.', ''); // Composition api isn't a class we don't need "this." here
+      return isTs && typeParameter ? `ref<${typeParameter}>(${cleanCode})` : `ref(${cleanCode})`;
+    },
```

Comment:
```suggestion
```

Reply:
oh sorry i meant to suggest reverting

```suggestion
    valueMapper: (code, _, typeParameter) =>
      isTs && typeParameter ? `ref<${typeParameter}>(${code})` : `ref(${code})`,
```